vue3/src/store/index.js

```diff
@@ -0,0 +1,53 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import createPersistedState from "vuex-persistedstate";
+
+Vue.use(Vuex);
+
+
+
+const store = new Vuex.Store({
+
+    plugins: [createPersistedState({ storage: window.sessionStorage })],
+
+    state: {
+        todos: [
+
+        ]
+    },
+    getters: {
+        todos: state => state.todos,
+        doneTodos: state => {
+            return state.todos.filter(todo => todo.status)
+        },
+        currentTodos: state => {
+            return state.todos.filter(todo => !todo.status)
+        },
+
+    },
+    mutations: {
+        doAdd(state, comment) {
+            state.todos.push({
+                id: state.todos.length,
+                comment: comment,
```

Comment:
オブジェクトはkeyとvalueが同じ文字の場合、以下のように省略できます。
```suggestion
                comment,
```